packages/@okta/vuepress-site/docs/reference/hooks-best-practices/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Hooks best practices
+excerpt: >-
+  Understand the best practices to implement and secure Okta Event Hooks or Inline Hooks.
+---
+
+# Hooks best practices
+
+Event Hooks and Inline Hooks are outbound calls from Okta to an external service. These outbound calls integrate custom functionality into your Okta processes. See [Event Hooks](/docs/concepts/event-hooks) and [Inline Hooks](/docs/concepts/inline-hooks) for additional concept information.
+
+The following sections review best practices to implement and secure Okta Event Hooks or Inline Hooks.
+
+## Secure your Hook endpoint
+
+To prevent a malicious actor from making requests to the endpoint where your Okta Hooks are sent, use the following best practices:
+
+* Configure Okta to send an authentication header in the Hook and validate it in every request by one of two ways:
+
+  * Using [HTTP Basic Authentication](/books/api-security/authn/api-authentication-options/#http-basic-authentication). When activating and enabling your Hooks on the Okta org, set the **Authorization field** as `authorization` and the **Authentication secret** in the Base64 `user:password` format.
```

Comment:
It's common practice to capitalize the 1st leader of a header field so can we change this to:

> set the Authorization field as "Authorization"